feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -499,14 +499,21 @@ func TestSoftwareVersion(t *testing.T) {
 		if v, ok := parent.Val(); ok {
 			got := gnmi.Get(t, dut, gnmi.OC().Component(v).Type().State())
 
+			switch dut.Model() {
 			// Arista_7280 OC component EOS has parent type Chassis
-			if dut.Model() == "DCS-7280CR3K-32D4" {
+			case "DCS-7280CR3K-32D4":
 				if got == chassisType {
 					t.Logf("Got a valid parent %v with a type %v for the component %v", v, got, os)
 				} else {
 					t.Errorf("Got a parent %v with a type %v for the component %v, want %v", v, got, os, chassisType)
 				}
-			} else {
+			case "CISCO-8808":
```

Comment:
Is there a better way we can do this instead of a switch case with `dut.Model`? 

Deviations can now be enabled for a specific vendor+hardware model so perhaps we could add a deviation here instead of a switch case with `dut.Model()` ?

Reply:
Thanks @prinikasn. Do you have any example of a metadata.proto that targets specific hardware model for additional deviations? We have a set of deviations that apply to all models and one additional deviation that only applies to CISCO-8808. Having two 'platform' entries with the same vendor gives
```
Error looking up deviations: cannot have more than one match within platform_exceptions fields platform:{vendor:CISCO}
```